apps/backoffice/src/utils/string-util.ts

```diff
@@ -26,3 +28,20 @@ export const isValidPath = (asPath: string) =>
   // Implement path validation logic, i.e. check that it does not contain dangerous characters
   /* eslint-disable no-useless-escape */
   /^[a-zA-Z0-9\-_\/]+$/.test(asPath);
+
+export const parseStringToNumberFunction = (
+  variable: null | string,
```

Comment:
**question:** why not “force” the input param to only string type and let the caller control the `null` value? Also, it is currently very coupled to an HTTP response (see the specific error it raises on `null`)

Reply:
[f086d9d](https://github.com/pagopa/io-services-cms/pull/1007/commits/f086d9df4c825ea56b7384912c699d9f2df87b56)